plugins/woocommerce-admin/client/products/fields/attribute-field/attribute-field.tsx

```diff
@@ -57,8 +60,17 @@ export const AttributeField: React.FC< AttributeFieldProps > = ( {
 		null | string
 	>( null );
 
-	const CANCEL_BUTTON_EVENT_NAME =
-		'product_add_attributes_modal_cancel_button_click';
+	const isOnlyForVariations = attributeType === 'for-variations';
```

Comment:
Obviously this sort of a boolean flag won't work if we decide to support `any` for the `attributeType` prop.

Reply:
Same comment as above, this forces knowledge around the consumer.  If we want to go  this route, we could just create a prop that was `isOnlyForVariations` directly instead.  

But this seems limiting and removing knowledge around the type of attribute altogether feels like the best path forward.